appcues/src/main/java/com/appcues/statemachine/StateMachine.kt

```diff
@@ -197,7 +197,7 @@ internal class StateMachine(
             // we are sitting on the last step if we get here - check if this is a continue with offset 1 (default continue)
             if (action.stepReference is StepOffset && action.stepReference.offset == 1) {
                 // for a continue -> next, on the last step, treat this the same as a close
-                val endExperience = EndExperience(destroyed = false)
+                val endExperience = EndExperience(markComplete = true, destroyed = false)
```

Comment:
in this case, we know we are on the last step as we are handling the continue action on last step to close - so always `true`